### ISSUE TITLE
false positive release duplicate for multi context

### ISSUE THREAD

**Issue by zhilyaev** [helmwave/helmwave#910]
```yaml
releases:
  # {{ range $c := $clusters }}
  - name: {{ $name }}
    <<: *options
    chart: oci://registry-1.docker.io/bitnamicharts/bla-bla
    context: {{ .context}}
  # {{- end }} finish
```

**Comment by zhilyaev** [comment]
extend uniqname like release@ns@context

**Comment by r3nic1e** [comment]
That's backward incompatible changes, need to specifically note that in changelog

**Comment by zhilyaev** [comment]
fixed in #914
